app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -174,6 +188,14 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardPopup.Listener {
             if (oldMessagesCount == 0 && dataSet.isNotEmpty()) {
                 recycler_view.scrollToPosition(0)
             }
+            contentToShare?.let {
```

Comment:
Don't know if this is the right place to "plug" this...
Setting contentToShare as null seems like a code smell

Reply:
I know you are waiting the message list to be populated, but maybe we need a new callback on the View, to indicate the first messages are loaded.